common/src/main/java/com/microsoft/identity/common/internal/fido/AuthFidoChallengeHandler.kt

```diff
@@ -190,10 +191,10 @@ class AuthFidoChallengeHandler (
         if (exception != null) {
             span.recordException(exception)
             span.setStatus(StatusCode.ERROR)
-            respondToChallenge(submitUrl, exception.javaClass.name + ": " + exception.message, context, span)
+            respondToChallenge(submitUrl, PASSKEY_PROTOCOL_ERROR_PREFIX_STRING + exception.javaClass.name + ": " + exception.message, context, span)
         } else {
             span.setStatus(StatusCode.ERROR, errorMessage)
-            respondToChallenge(submitUrl, errorMessage, context, span)
+            respondToChallenge(submitUrl, PASSKEY_PROTOCOL_ERROR_PREFIX_STRING + errorMessage, context, span)
```

Comment:
>PASSKEY_PROTOCOL_ERROR_PREFIX_STRING + errorMessage [](http://example.com/codeflow?start=42&length=51)

nit: This can be done private util method like getErrorAssertion()

(not imp to merge this change)

Reply:
Something like this? : eb0f6d2bf90fe805f24743fb79c5276e431bc822